src/readbin/byteweight_util.ml

```diff
@@ -209,6 +229,22 @@ module Cmdline = struct
     let doc = "Print symbol's size." in
     Arg.(value & flag & info ["print-size"; "s"] ~doc)
 
+  let output_format : [`text | `sexp] Term.t =
+    let enums = ["text", `text; "sexp", `sexp] in
+    let doc = sprintf "The output format. %s" @@ Arg.doc_alts_enum enums in
+    Arg.(value & opt (enum enums) `sexp & info ["output_format"; "f"] ~doc)
+
+  let tool : [`BW | `SymTbl] Term.t =
+    let enums = ["BW", `BW; "SymTbl", `SymTbl] in
```

Comment:
what is the reason for use this unreadable acronyms? Why not `byteweight` `symbol-table`. Your interface is enough complicated for such trivial tool.